download_and_install/compiling_from_source.md

```diff
@@ -0,0 +1,242 @@
+        protobuf \
+        zlib
+
+Python dependencies
+-------------------
+
+If you wish to build the Python library, additional libraries are required:
+
+    pip3 install --user \
+        numpy \
+        protobuf
+
+If the pip3 command does not exist, install python3-pip from your package manager and try again.
+
+Build
+=====
+
+If this is the first time compiling with this download, generate the build system:
+
+    autoreconf -i
+
+OLA uses autotools for building. Run
+
+     ./configure --help
+
+to see all options. The most popular option is `--enable-python-libs` to build the Python client module. If you want to
+use the RDM responder tests, add `--enable-rdm-tests`.
+
+Once you’ve decided on the options, it’s time to build OLA.
+
+    ./configure <your options>
+    make -j$(nproc)
+    make check
+    sudo make install
+
+Mac users should run use `make -j$(sysctl -n hw.logicalcpu)` instead of `make -j$(nproc)`.
+
+Finally, run `sudo ldconfig` to make new libraries available.
+
+Device drivers
+==============
+
+Note that, for some devices, it is necessary to install drivers for OLA to work with them. For example, the
+[Open DMX USB](http://opendmx.net/index.php/Open_DMX_USB "Open DMX USB") device needs an additional kernel module that
+could be built using the instructions on
+[LLA\_and\_Q\_Light\_Controller\_Ubuntu\_Tutorial](http://opendmx.net/index.php/LLA_and_Q_Light_Controller_Ubuntu_Tutorial)
+. For other devices, refer to the corresponding device page on the wiki.
+
+Known Issues
+============
+
+If you get an error like the following:
+
+    /bin/sh ./libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.   -I/opt/local/var/macports/software/protobuf-cpp/2.0.3_0/opt/local/include/  -g -O2 -c -o ltdl.lo ltdl.c
+    ./libtool: line 464: CDPATH: command not found
+    /Users/simonn/lighting/lla/libltdl/libtool: line 464: CDPATH: command not found
+    /Users/simonn/lighting/lla/libltdl/libtool: line 1142: func_opt_split: command not found
+    libtool: Version mismatch error.  This is libtool 2.2.6, but the
+    libtool: definition of this LT_INIT comes from an older release.
+    libtool: You should recreate aclocal.m4 with macros from libtool 2.2.6
+    libtool: and run autoconf again.
+
+Your system uses a different version of libtool. Run:
+
+     libtoolize --ltdl -c -f
+
+and then start from the autoreconf step again.
+
+If you should get the following error try to fix it with one of
+[two available solutions](https://groups.google.com/group/open-lighting/msg/72060f6327d30df6):
+
+    Rpc.pb.cc: In copy constructor 'ola::rpc::RpcMessage::RpcMessage(const ola::rpc::RpcMessage&)': 
+    Rpc.pb.cc:143: error: base class 'class google::protobuf::Message' should be explicitly initialized in the copy constructor
+
+You should be able to prevent this
+by [editing `./src/Makefile.am`](https://groups.google.com/group/open-lighting/msg/c6d86d03dd74ed5b), removing `-Werror`
+and then start from the autoreconfig step again.
```

Comment:
I think this can just be adding `--disable-fatal-warnings` to configure args now.